sky/resources.py

```diff
@@ -944,9 +941,20 @@ def less_demanding_than(self,
             if other.disk_tier is None:
                 return False
             if self.disk_tier != other.disk_tier:
-                types = ['low', 'medium', 'high']
-                return types.index(self.disk_tier) < types.index(
-                    other.disk_tier)
+                if self.cloud is not None:
+                    self_tier = self.cloud.translate_disk_tier(self.disk_tier)
+                else:
+                    self_tier = self.disk_tier
+                if other.cloud is not None:
+                    other_tier = other.cloud.translate_disk_tier(
+                        other.disk_tier)
+                else:
+                    other_tier = other.disk_tier
+                # best disk tier without knowledge of cloud cannot be compared
+                if resources_utils.DiskTier.BEST in [self_tier, other_tier]:
```

Comment:
Does it mean we cannot `sky launch -c existing-cluster --disk-tier best`, even if the existing cluster was launched with `best` disk tier?

Reply:
Oh that make sense... Need some new design